lib/ood_core/job/adapters/linux_host/templates/script_wrapper.erb.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
-hostname
+hostnames=$(hostname -A)
+hostnameFound=false
+for host in $SSH_HOSTS
+do
+    if [[ " ${hostnames[@]} " =~ " ${host} " ]]; then
+        hostnameFound=true
+        hostname=$host
+    fi
+done
+
+if [ ! $hostnameFound ]; then
+    echo >&2 "ERROR: The specified host is not in the list of ssh hosts configured in this cluster config [${SSH_HOSTS[@]}]. The specified hosts determined by running 'hostname -A' [${hostnames[@]}] on the target host must match one of the configured ssh hosts" 
+    exit 1
+fi
```

Comment:
This if statement doesn't work. I mean it never evaluates to false. 

I think if you get rid of the boolean and just check to see if `hostname` ever got set is a safer test. Bash/shells can be funny about booleans so I usually just write a string and is it's existence as the boolean value.
```bash
if [[ -n "${hostnameFound+x}" ]]; then
    # ...
fi
```

Also, since it's already in an erb, we should probably have a translatable error message.

Reply:
What do you mean by 'translatable error message'?

Reply:
Now that I actually say it, I don't think ood_core deals with il8n. Rails does, which this is not, so you can disregard. 

But what I'd meant was Rails' I18n library so that messages can default to english but we'd have the ability to translate it to another language later through configuration.